### DISCUSSION TITLE
How to install .exe file in gui environment

### DISCUSSION THREAD

**Discussion by ghost** [termux/termux-packages#11717] [General]
Please give commands to install it

**Comment by ghost** [comment]
Normalley, .exe files are windows executables. They will not work on Termux, which is based on Linux. So, the best available command may be,  "Buy yourself a computer with Windows operating system."

**Reply by USLTD** [reply]
What about Box86/Box64 + Wine mix? Haven't tried compiling them in vanilla (no proot) Termux environment but in proot they work without problem. Except that it is little slow and not every .exe is supported

**Comment by IntinteDAO** [comment]
Well, in theory you can use ARM64 Windows apps on Termux using Wine, but most (99.9%) Win Apps are not for ARM64

If you want to use Wine application, you can use Exagear for that, but it's risky, cause Exagear is not supported and you can only download it from strange websites
